sites/cheerpj/src/content/docs/23-licensing.md

```diff
@@ -2,48 +2,68 @@
 title: Licensing
 ---
 
-CheerpJ is free to use for personal projects, most non-commercial applications, and for technical evaluations.
+CheerpJ Core is distributed at no cost with the **CheerpJ Community License**, which makes it free to use for _personal projects_, _free_ and _open-source software_, and for _technical evaluations_.
```

Comment:
I would keep the express as FOSS project. I want to make sure the "free" part of FOSS is not misunderstood as "not payed, even if commercial"